package.json

```diff
@@ -227,22 +267,22 @@
     ]
   },
   "lint-staged": {
-    "*.json": "yarn prettier-write --parser json",
-    "*.ts": "yarn prettier-write --parser typescript"
+    "*.json": "yarn prettier --write",
+    "*.ts": "yarn prettier --write"
```

Comment:
**Non-blocking suggestion**: We should look at adding `eslint` in here too like [this example](https://github.com/okonet/lint-staged#running-multiple-commands-in-a-sequence).

Reply:
I was actually having a 2nd thought on these post-commit hooks... The fact that the code can be magically changed behind my back after commit was not a conform to me, especially during the early days when I first involved in this project... now we have ci guard on both lint and format, I almost feel we could get rid of these post-commit hooks so we know what we see in vscode is what will show up in the github PR... 

This could be one of those subjective preferences without clear right or wrong... do you guys have a strong preference?